src/test/scala/show/ShowTests.scala

```diff
@@ -0,0 +1,35 @@
+package show
+
+import me.chuwy.otusbats.Show._
+import org.scalatest.flatspec.AnyFlatSpec
+
+class ShowTests extends AnyFlatSpec {
+
+  "Check int show" should "return proper string" in {
+    assert(
+      5.show === "5"
+    )
+  }
+
+  "Check list of ints show" should "return proper string" in {
+    assert(
+      List(1, 2, 3).show === "123"
+    )
+  }
+  // "Check list of ints mkString" should "return proper string" in {
+  //   assert(
+  //     List(1, 2, 3).mkString_(", ", "{ ", " }") === "{ 1, 2, 3 }"//This does not compile
```

Comment:
Тут к сожалению компилятор просит помочь ему немного:

```suggestion
       List(1, 2, 3).mkString_[Int](", ", "{ ", " }") === "{ 1, 2, 3 }"//This does not compile
```

Такое часто бывает и в целом есть способы этого избежать и те же cats применяют много сложных техник чтобы сделать клиентский код более лаконичным, но а) это всё сложно; б) в Scala 3 ситуация гораздо лучше; в) как начинающим нам важно знать что при любых непонятных ошибках мы в первую очередь подсовываем правильные типы

Reply:
Спасибо!!!